libs/blocks/global-navigation/global-navigation.js

```diff
@@ -650,7 +650,9 @@ class Gnav {
   };
 
   loadSearch = () => {
-    if (this.blocks?.search?.instance) return null;
+    const instanceAlreadyExists = !!this.blocks?.search?.instance;
+    const searchNotInContent = !this.searchPresent();
+    if (instanceAlreadyExists || searchNotInContent) return null;
```

Comment:
minor: can directly use `!this.searchPresent()` in `if` condition instead of creating variable